common/djangoapps/student/views.py

```diff
@@ -1167,6 +1167,8 @@ def login_user(request, error=""):  # pylint: disable-msg=too-many-statements,un
 @social_utils.strategy("social:complete")
 def login_oauth_token(request, backend):
     """
+    DEPRECATED - Note that this endpoint is deprecated.  Please use AccessTokenExchangeView instead.
```

Comment:
I don't think we want to fill our logs with warnings for this. Old versions of our mobile app will still call this until they are upgraded, and it doesn't seem like something we need to warn about just yet.